TileDB.CSharp/TileDB.CSharp.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>TileDB.CSharp</id>
+    <version>2.4.4</version>
```

Comment:
Should we version the numbers separate from core?

Reply:
It tries to make one-to-one correspondence between core and its wrapper. so the wrapper just use the same version number of core